libs/pmp-web/shared/ui-prs-page/src/lib/components/prs-page/prs-page.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="page__container">
+  <pmp-navbar class="page__navbar"></pmp-navbar>
+  <pmp-sidebar class="page__sidebar"></pmp-sidebar>
+  <router-outlet></router-outlet>
```

Comment:
We spoke that it is not a good practice to have router outilet in ui component. This was fully intentional to copy that part of html in every module.